config/recurring.yml

```
@@ -13,3 +13,7 @@ production:
   clear_solid_queue_finished_jobs:
     command: "SolidQueue::Job.clear_finished_in_batches(sleep_between_batches: 0.3)"
     schedule: every hour at minute 12
+  placemetn_preferences_reminder_job:
+    class: "PlacementPreferenceReminderJob"
+    queue: "mailers"
+    schedule: "0 9 1 * * Europe/London"
```

```suggestion
  placement_preferences_reminder_job:
    class: "PlacementPreferenceReminderJob"
    queue: "mailers"
    schedule: "0 9 1 * * Europe/London"
```

Typo aside, could we remove this for now so we're not immediately sending emails from the service as it goes live please? I know it was requested and everything else looks great but we can add the schedule back later :)

---

Removed